src/java.desktop/share/classes/javax/swing/SwingUtilities.java

```diff
@@ -1810,6 +1810,13 @@ else if (!stayNull && event.getKeyChar() != KeyEvent.CHAR_UNDEFINED) {
         action.actionPerformed(new ActionEvent(sender,
                         ActionEvent.ACTION_PERFORMED, command, event.getWhen(),
                         modifiers));
+        if (event.getSource() instanceof JToggleButton) {
+            JToggleButton tb = (JToggleButton)event.getSource();
```

Comment:
Please use a pattern variable here:
```suggestion
        if (event.getSource() instanceof JToggleButton tb) {
```